pennylane/math/single_dispatch.py

```diff
@@ -120,6 +120,20 @@ def _take_autograd(tensor, indices, axis=None):
 ar.autoray._SUBMODULE_ALIASES["tensorflow", "diag"] = "tensorflow.linalg"
 
 
+tf_fft_functions = [
+    "fft",
+    "ifft",
+    "fft2",
+    "ifft2",
+]
+
+
+for fn in tf_fft_functions:
+    ar.autoray._SUBMODULE_ALIASES["tensorflow", "fft." + fn] = "tensorflow.signal"
+
+ar.autoray._FUNC_ALIASES["tensorflow", "fft.fft2"] = "fft2d"
+ar.autoray._FUNC_ALIASES["tensorflow", "fft.ifft2"] = "ifft2d"

```

Comment:
This now registers that the TF functions have different names

Reply:
This is to make the trig function names consistent across all interfaces so you can just so `qml.math.func`, right? Do you mind adding `atan2` to the list? :pray: I ran into this issue earlier in #1384 with `acos` and `atan2` :sweat_smile:

Reply:
Yep! @glassnotes feel free to add this directly to #1394 --- this PR will likely not be merged for a while, and shouldn't block yours in the meantime 😆 

You will need to add the following two lines:

```python
ar.autoray._SUBMODULE_ALIASES["tensorflow", "arctan2"] = "tensorflow.math"
ar.autoray._FUNC_ALIASES["tensorflow", "arctan2"] = "atan2"
```

The first registers that TF makes the `arctan2` function accessible under the `math` submodule, the second indicates the name change.

(re: `acos` - I can see that one is already registered? What issue were you having?)

Reply:
Ah it ended up being fine and I just needed to add `arctan2` (to both `torch` and `tf`). Thanks! :star2: